components/math/multiplicationConnect/Modal.tsx

```diff
@@ -0,0 +1,70 @@
+import React from "react";
+
+/* todo: Modal following WDS tutorial using React portals
```

Comment:
i'm looking forward to seeing how this is implemented

Reply:
Implemented, you can check out the branch! Didn't use Portals though, just ended up using an overlay and changing the z-index of elements to make sure they appear over/under.